_docs-sources/foundations/grunty/architecture.mdx

```diff
@@ -0,0 +1,94 @@
+# Architecture
+
+The Grunty GitHub App is a GitHub App introduced to help reduce the burden of integrating Gruntwork products to GitHub resources. The app is designed to be lightweight and flexible, providing a simple way to get started with Gruntwork products.
+
+## Overview
+
+There are three major components to keep in mind when working with the Grunty GitHub App:
+
+1. **GitHub App**: The Grunty GitHub App itself, which is installed in your GitHub organization.
+2. **Grunty API**: The API that interacts with the GitHub API on behalf of the Grunty GitHub App.
+3. **Clients**: Clients that interact with the Grunty API to perform operations on GitHub resources.
+
+## GitHub App
+
+The Grunty GitHub App is the principal that Gruntwork products will utilize to interact with GitHub resources. The app can either be installed on a per-repository basis or on an entire organization.
```

Comment:
What does "principal" mean? Can you clarify in the text? In fact, my understanding is that Gruntwork doesn't "interact" with the GitHub app, but rather that the GitHub app gives the Gruntwork app credentials it can use to call the GitHub API.